ucfcs/GrowthPlus
# Scenario Game Content

-Added Scenario Game Content, the content has a question, three options, and an answer (this means we will only have 3 fish in Game 1, instead of 4)
-Styled fragments, quiz timer, and added new unit images 
-Added top bar star to game intro screen and game screen 


Test:
- wipe data on phone and make a new child
- Check realm data to see if scenario game has a list of 40 contents
- Check game tile to see if star is correctly added to top bar

Closes #133
